openpype/hosts/maya/plugins/load/load_audio.py

```diff
@@ -59,8 +59,24 @@ def update(self, container, representation):
         assert audio_nodes is not None, "Audio node not found."
         audio_node = audio_nodes[0]
 
+        current_sound = cmds.timeControl(
+            mel.eval("$tmpVar=$gPlayBackSlider"),
```

Comment:
By the way - this `mel.eval` query should really be: `$gPlayBackSlider=$gPlayBackSlider`

The reason for that is that if you use `$tmpVar` more than once you might end up getting MEL "variable redeclaration errors" since if you also use `$tmpVar` for retrieving other global variables that if those other variables have a different type (say integer or float instead of string) then MEL will complain because it's statically typed and thus warns on the variable declaration.

Reply:
```suggestion
            mel.eval("$gPlayBackSlider=$gPlayBackSlider"),
```